system-addon/content-src/components/Search/Search.jsx

```diff
@@ -48,13 +54,15 @@ class Search extends React.Component {
         <span className="sr-only"><FormattedMessage id="search_web_placeholder" /></span>
       </label>
       <input
+        autoFocus="autofocus"
```

Comment:
we want this only for about:home

Reply:
and only on first visible ?

Reply:
Ah yes you're right, I'll just remove this because it's broken anyways and there's a ticket to so autofocus anyways #2007